keps/sig-cli/3805-ssa-default/README.md

```diff
@@ -129,36 +134,64 @@ The meaning of `auto` goes as follows:
 - If the resource already exists but doesn't have the `last-applied`
   annotation, the resource is server-side applied
 
+We are also planning on switch the `--server-side` flag to `true` in
+three releases, by adding a warning message (and a blog-post) in kubectl
+1.27 to let people know that the change will happen.
```

Comment:
Can you please mark this as provisional as well and remove the reference to a specific release? I don't think we can warn without having the deprecation approved, and that is not something we'll be able to get adequate attention for in the remaining hours.